src/openrct2/ride/Ride.cpp

```diff
@@ -5522,7 +5533,8 @@ bool Ride::NameExists(const std::string_view& name, ride_id_t excludeRideId)
     {
         if (ride.id != excludeRideId)
         {
-            ride.FormatNameTo(formatArgs);
+            Formatter ft(reinterpret_cast<uint8_t*>(formatArgs));
```

Comment:
Maybe `formatArgs` above should just be changed to `uint8_t` ? Seems like this is more correct than the cast in here and you already did this at the top of the diff, on `EditorObjectiveOptions.cpp `